tissuumaps/static/js/utils/overlayUtils.js

```diff
@@ -651,7 +651,9 @@ overlayUtils.areAllFullyLoaded = function () {
 overlayUtils.waitLayersReady = async function () {
     var op = tmapp["object_prefix"];
     await new Promise(r => setTimeout(r, 200));
-    while (!tmapp[op + "_viewer"].world || tmapp[op + "_viewer"].world.getItemCount() != tmapp.layers.length) {
+    while (!tmapp[op + "_viewer"].world || 
+        ((tmapp[op + "_viewer"].world.getItemCount() != tmapp.layers.length) &&
+        !(tmapp[op + "_viewer"].world.getItemCount() == 1 && tmapp.layers.length == 0))) {
```

Comment:
This could maybe be simplified to:
```suggestion
    while (!tmapp[op + "_viewer"].world || 
        tmapp[op + "_viewer"].world.getItemCount() < tmapp.layers.length) {
```